raiden/transfer/channel.py

```diff
@@ -1574,6 +1639,50 @@ def send_lock_expired(
     return []
 
 
+def is_withdraw_expired(
+    block_number: BlockNumber, expiration_threshold: BlockNumber
+) -> SuccessOrError:
```

Comment:
The error message from this function is being ignored, so it can be just a normal predicated.

Note: We need a better name for functions returning `SuccessOrError`. Ideas? I would prefer to have functions starting with `is_` only returning booleans.

Reply:
Side note: IMO `SuccessOrError` should be removed in favor of `Optional[ErrorMSG]`

Reply:
`Optional[ErrorMsg]` sounds best IMO. Created an issue for this: https://github.com/raiden-network/raiden/issues/4296